src/blog/get-a-birds-eye-view-with-truffle-teams-new-dashboard.md

```diff
@@ -0,0 +1,43 @@
+<figure>
+  <img class="mb-4" src="/img/blog/get-a-birds-eye-view-with-truffle-teams-new-dashboard/blog-header.jpg" alt="Truffle Teams Dashboard" style="width:100%">
+</figure>
+
+As the operational needs of your dapp grow in complexity, it helps to be able to choose an area of focus. More incoming data begets a greater need to gain insights and properly segment that data to prevent overload. Wouldn’t it be nice to see an overview of a particular deployed instance of your application with the ability to drill down into the parts you need to see in more detail? This is where the dashboard comes in!
+
+<p class="alert alert-warning">💬 The Dashboard feature could expand in many different directions and we need your help to prioritize what matters most for your dapps. In that spirit we’ve released this first version quickly to get the conversation started. Please comment in the <a href="https://join.slack.com/t/truffle-community/shared_invite/zt-8wab0bnl-KcugRAqsY9yeNJYcnanfLA">Truffler Slack workspace</a>, <a href="mailto:info@trufflesuite.com">email us</a>, reach out any way you can--we’re excited to hear from you.</p>
+
+We first asked ourselves “what is a dashboard”? and surprisingly, got many different answers! We arrived at this definition: **A dashboard is a place to see an overview of your application, along with ways to drill down and get further context on more narrow slices.**
+
+The first release focuses mostly on the high-level, but we plan to increase the drilldown capabilities as the feature evolves.
+
+## A Bird’s Eye View
+
+<figure>
+  <img class="mb-4 figure-shadow" src="/img/blog/get-a-birds-eye-view-with-truffle-teams-new-dashboard/truffle-teams-dashboard.png" alt="Truffle Teams Dashboard" style="width:100%">
+  <figcaption class="text-center font-italic">The dashboard page with a deployment selected</figcaption>
+</figure>
+
+In this first version of the dashboard, you can pin a deployment for quick insights and visualization. Some of this information we haven’t surfaced before, such as the deployment’s cost and the number of unique addresses interacting with a deployment’s contracts.
+
+<div class="mt-12 text-center">
+  <a class="btn btn-truffle mt-3" href="https://my.truffleteams.com/" target="_blank">CHECK OUT THE NEW DASHBOARD</a>
+</div>
+
+## Looking Ahead
+
+Here’s a bit of a teaser for you to get an idea of where this is going:
+
+* A filter option to specify exactly which contracts are being displayed in the transaction graph
+* A list of contracts providing the ability to hop into monitoring a specific contract instance or...
+* Quickly interact with those contracts via our upcoming Contract Manager
+
+We want Truffle Teams to be the most effective DevOps tool in the blockchain space for both new and existing teams. Let us know what you think about this integration solution, and if your team has other needs we haven’t met yet.
```

Comment:
Maybe instead of "We want Truffle Teams to be the most effective..." could be "We strive for Truffle Teams to be the most effective". Conveying that it's more than a desire.

Reply:
Love it--updated!